Makefile

```diff
@@ -6,7 +6,6 @@ all: $(BINS)
 $(BINS):
 	make -C $@
 
-include rules.mk
```

Comment:
i think we still need this or we need to move the contents from rules.mk into the Makefile